app/javascript/react/screens/App/Plan/helpers.js

```diff
@@ -115,3 +120,19 @@ export const incompleteCancellationTasks = (requestWithTasks, actions, tasksForC
   const completedTasksOfPlan = tasksOfPlan.filter(task => task.state === 'finished');
   return removeCancelledTasksFromSelection(tasksForCancel, completedTasksOfPlan);
 };
+
+export const parseComplexErrorMessages = complexErrorMessage => {
+  if (V2V_BACKEND_ERROR_MESSAGES.V2V_DOWNLOAD_LOG_STATUS_MESSAGES.exec(complexErrorMessage) !== null) {
```

Comment:
When switching to this branch, the error notification seems to stop working.  By any chance, should the conditional be checking

```js
V2V_BACKEND_ERROR_MESSAGES.DOWNLOAD_LOG_ERROR_NO_HOST.exec(...) !== null
```

Reply:
Ah, I think Mike is right here... I admit I hadn't tested it again after the second commit. I need to break that habit.

Reply:
Thanks @michaelkro @mturley -- I have made the above adjustment.

... and sorry for the hiatus on this PR (was tied up in other things)